src/log.py

```diff
@@ -87,20 +87,21 @@ def _update_log_parent_symlinks():
 
     one_year_ago = time.time() - 365 * 24 * 3600
 
-    logs = glob.iglob(OPERATIONS_PATH + "*" + METADATA_FILE_EXT)
+    logs = glob.iglob("*" + METADATA_FILE_EXT, root_dir=OPERATIONS_PATH)
     for log_md in logs:
-        if os.path.getctime(log_md) < one_year_ago:
+        log_file = os.path.join(OPERATIONS_PATH, log_md)
+        if os.path.getctime(log_file) < one_year_ago:
             # Let's ignore files older than one year because hmpf reading a shitload of yml is not free
             continue
 
         name = log_md[: -len(METADATA_FILE_EXT)]
         parent_symlink = os.path.join(OPERATIONS_PATH, f".{name}.parent.yml")
-        if os.path.islink(parent_symlink):
+        if (os.path.islink(log_file) and os.path.realpath(log_file) == "/dev/null") or os.path.islink(parent_symlink):
```

Comment:
We probably don't need this change then?

Reply:
Hmmyeah i'm not sure to understand what was the intent ... naively we're good to go with just the stuff that fixes the confusion about relative vs absolute paths